server/odc-migrate/src/main/resources/migrate/common/V_4_3_2_1__add_integration_git.sql

```diff
@@ -0,0 +1,34 @@
+CREATE TABLE IF NOT EXISTS `integration_git_repository` (
+  `id` bigint(20) NOT NULL AUTO_INCREMENT,
+  `create_time` datetime NOT NULL DEFAULT CURRENT_TIMESTAMP,
+  `update_time` datetime DEFAULT CURRENT_TIMESTAMP ON UPDATE CURRENT_TIMESTAMP,
+  `creator_id` bigint(20) NOT NULL COMMENT 'creator user id, references iam_user.id',
+  `organization_id` bigint(20) NOT NULL COMMENT 'organization id, references iam_organization.id',
+  `project_id` bigint(20) DEFAULT NULL COMMENT 'project id, references collaboration_project.id',
+  `description` varchar(512) DEFAULT NULL COMMENT 'description',
+  `name` varchar(512) NOT NULL COMMENT 'repository name',
+  `provider_type` varchar(32) NOT NULL COMMENT 'git provider type',
+  `provider_url` varchar(128) NOT NULL COMMENT 'git provider url',
+  `ssh_url` varchar(512) NOT NULL COMMENT 'ssh clone url, like git@github.com:xxx/xxx.git',
+  `clone_url` varchar(512) NOT NULL COMMENT 'https clone url, like https://github.com/xxx/xxx.git',
+  `email` varchar(128) NOT NULL COMMENT 'user email for commit',
+  `token` varchar(512) NOT NULL COMMENT 'person access token',
+  `salt` varchar(32) DEFAULT NULL COMMENT 'used to connect the random value used by the encryption and decryption algorithm of the secret field',
+  PRIMARY KEY (`id`),
+  KEY `integration_git_repository_project_id` (`project_id`)
+) COMMENT = 'integration git repository';
+
+CREATE TABLE IF NOT EXISTS `integration_git_repository_stage` (
```

Comment:
organization_id column always required for multi tenant architecture

Reply:
done